plugins/archlinux/archlinux.plugin.zsh

```diff
@@ -191,5 +193,7 @@ if (( $+commands[yay] )); then
   alias yamir='yay -Syy'
   alias yaupd="yay -Sy"
   alias upgrade='yay -Syu'
+  alias yacun='yay -Yc'
+  alias yacf='yay -Scc'
```

Comment:
You should document these in the plugin README.

Reply:
ok, i do this.